stations.csv

```diff
@@ -70508,3 +70508,4 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 74649;Paestum Santa Venere;paestum-santa-venere;8314556;;40.412770;15.010350;19697;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8314556;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;パエストゥム;;;;Пестум;;;帕埃斯图姆;urn:trainline:public:nloc:csv74649;
 74650;Paestum Parco Archeologico;paestum-parco-archeologico;8314555;;40.421390;15.002960;19697;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8314555;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;パエストゥム;;;;Пестум;;;帕埃斯图姆;urn:trainline:public:nloc:csv74650;
 74651;Mostoles El Soto;mostoles-el-soto;7135607;;40.330971;-3.882441;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;35607;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74651;
+74652;Bari Bus;bari-bus;;;41.116939;16.870599;22157;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;BAX;t;;f;;f;;f;;f;;f;f;;;;;;;;;;;;バーリ;바리;;;Бари;;;巴里;urn:trainline:public:nloc:csv74652;
```

Comment:
Shouldn't this be called Bari? I don't see anything named like that on openstreetmap
![image](https://github.com/trainline-eu/stations/assets/4478372/771903ee-3b49-4840-a08f-c415fece08a3)